### PR TITLE
Supervisor admins

### PR DESCRIPTION
**Pull request by sveredyuk**
Add new level to admin - supervisor!

### CODE REVIEW

File: spec/factories/user_factory.rb [hunk 1]
```diff
@@ -10,6 +10,11 @@
       admin true
     end
 
+    trait :supervisor do
+      admin      true
```
Comment (denys-medynskyi) [inline spec/factories/user_factory.rb:14]: Admin and supervisor at the same time?
It is more like a role and we should extract it to a separate table.
Do you agree? @sveredyuk

Comment (denys-medynskyi) [reply spec/factories/user_factory.rb:14]: Maybe policies?

Comment (sveredyuk) [reply spec/factories/user_factory.rb:14]: I agree
But we out of time. We need quick solution ASAP.

Comment (sveredyuk) [reply spec/factories/user_factory.rb:14]: @denysmedynskyi Sorry, but I need merge this to protect sensitive information from uncontrolled access by courses admins,

Comment (sveredyuk) [reply spec/factories/user_factory.rb:14]: And with new students we can develop role system and drop with quick workaround.

Comment (denys-medynskyi) [reply spec/factories/user_factory.rb:14]: I understand ASAP-ness.

**Pull request merged**